src/ui/Distance/DistanceResources.ts

```diff
@@ -315,11 +332,13 @@ export class DistanceResources extends Component {
   }
 
   private triggerDistanceNotSet(): void {
-    this.isPositionSet = false;
+    this.isPositionSet = true;
```

Comment:
You could rename this property to `isFirstPositionResolved`, since the position was not technically set :)